openslides/motions/static/js/motions/site.js

```diff
@@ -906,9 +907,21 @@ angular.module('OpenSlidesApp.motions.site', [
                 image_sources = image_sources.concat(tmp_image_sources);
             });
 
-            //post-request to convert the images. Async.
-            $http.post('/core/encode_media/', JSON.stringify(image_sources)).then(function (success) {
-                var converter = PdfMakeConverter.createInstance(success.data.images);
+            var image_map = [];
+            _.forEach(image_sources, function (image_source) {
+                image_map[image_source] = PDFLayout.imageURLtoBase64(image_source);
+            });
+
+            var image_promises = Object.keys(image_map).map(function( key ) {
+                return image_map[key];
+            });
+
+            Promise.all(image_promises).then(function(base64Str) {
```

Comment:
As I understand this ES5 has no Promise API so your code would not work in such an environment. We can later think about moving the codebase to ES2015 with babel or Typescript or something else ... It depends on our browser support policy. This I do not know at the moment. @emanuelschuetze What do you think.